### ISSUE TITLE
API module: $I->sendGET() does not work with absolute URL

### ISSUE THREAD

**Issue by akireikin** [Codeception/Codeception#2162]
I migrated to 2.1 and then it started. File upload tests were broken down. My app domain is `http://myapp.dev`, while uploaded files are stored `http://upload.myapp.dev`. Tests look like.

```php
// file upload (post request to domain http://myapp.dev)

// check success
$I->sendGET('http://upload.myapp.dev/doihiub982/document.pdf');
```

In 2.0.16 it works fine, but in 2.1 there is a `404` error. I have made a research and found out that problem is in `Guzzle 6` client. It requests `http://myapp.dev/doihiub982/document.pdf` instead of `http://upload.myapp.dev/doihiub982/document.pdf`.

**Comment by DavertMik** [comment]
Probably this is fixed in `2.1` branch and will be fixed in release 2.1.1 which comes in next hours. Stay tuned )

**Comment by Naktibalda** [comment]
Duplicate of #2149 

Fixed in 2.1.1
